Doc/README.rst

```diff
@@ -14,16 +14,17 @@ developers guide.
 Building the docs
 =================
 
-You need to have `Sphinx <http://sphinx-doc.org/>`_ installed; it is the toolset
-used to build the docs.  It is not included in this tree, but maintained
-separately and `available from PyPI <https://pypi.python.org/pypi/Sphinx>`_.
+The documentation can be built using the ``make`` tool.
+
+The process itself requires Python and `Sphinx <http://sphinx-doc.org/>`_. As
+long as you have a working Python interpreter available, the rest is handled
+automatically, so you don't have to install Sphinx yourself.
 
 
 Using make
 ----------
 
-A Makefile has been prepared so that on Unix, provided you have installed
-Sphinx, you can just run ::
+A Makefile has been prepared so that (on Unix) you can just run ::
```

Comment:
Hi @cjrh, Similar to line 95, I would mention changing into the Doc directory before running `make html` to assist those folks that are newer to Sphinx.